classes/Hook.php

```diff
@@ -1129,13 +1129,7 @@ private static function getHookModuleFilter()
             $serviceContainer = ContainerBuilder::getContainer('front', _PS_MODE_DEV_);
         }
 
-        try {
-            $hookModuleFilter = $serviceContainer->get(HookModuleFilter::class);
-        } catch (Exception $e) {
-            return null;
-        }
-
-        return $hookModuleFilter;
+        return $serviceContainer->get(HookModuleFilter::class);
```

Comment:
Hello @matthieu-rolland 
This method no longer returns null, can you please update the PhpDocs in this case ? Also add `@throws ServiceNotFoundException`